### DISCUSSION TITLE
How does this working with fixtures and purging behavior

### DISCUSSION THREAD

**Discussion by agennari** [RamyHakam/multi_tenancy_bundle#39] [General]
Hi,

Does anyone already have this bundle working with fixtures?
The issue is at the purger level

The purger is using the EntityManager (not tenant) and so, it's impossible to purge the Tenant dbs
I tried to make a custom purger, but at some doctrine levels, we can't change the EntityManager used and It still using EntityManager (main), not Tenant...

**Comment by RamyHakam** [comment]
Hi,
This is now supported! 🎉

You can use the new `#[TenantFixture]` attribute to define your tenant-specific fixtures and run them with the new `tenant:fixtures: load` command. 

It utilizes the correct TenantEntityManager and a custom purger under the hood — no manual overrides are necessary.

Give it a try! ✅
